talpid-core/src/split_tunnel/windows/mod.rs

```diff
@@ -346,34 +347,28 @@ impl SplitTunnel {
     pub fn clear_tunnel_addresses(&mut self) -> Result<(), Error> {
         self._route_change_callback = None;
 
-        Self::register_ips(
-            &*self.handle,
-            Ipv4Addr::new(0, 0, 0, 0),
-            None,
-            Ipv4Addr::new(0, 0, 0, 0),
-            None,
-        )
+        Self::register_ips(&*self.handle, Ipv4Addr::new(0, 0, 0, 0), None, None, None)
```

Comment:
Second argument can also be `None`?

Reply:
It probably should be. Anyway, this can be improved later. Leaving it like this for now.